Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -173,10 +173,17 @@ void MenuBar::AddViewMenu()
 void MenuBar::AddHelpMenu()
 {
   QMenu* help_menu = addMenu(tr("Help"));
+  QAction* website = help_menu->addAction(tr("Website"));
+  connect(website, &QAction::triggered, this,
+          [=]() { QDesktopServices::openUrl(QUrl(QStringLiteral("https://dolphin-emu.org/"))); });
```

Comment:
This is not 100% accurate, but: 
The action of opening a URL is asynchronous in respect to Dolphin, so it's in a lambda (which connect calls asynchronously)
If it was just a normal slot, then connect would re-enter the the Qt event loop, then call the slot, then execute the code.


Even if this isn't true, there's no side effects since this is:
1. A non essential operation
2. Not a time critical operation
3. The act of using a lambda itself doesn't affect Dolphin's state, only Qt's.


The part I know for a fact is a benefit is that no slot needs to be written to open the URL's. This is one of the textbook reasons that lambdas are useful.